lib/rubocop/cop/internal_affairs/undefined_config.rb

```diff
@@ -5,12 +5,22 @@ module Cop
     module InternalAffairs
       # Looks for references to a cop configuration key that isn't defined in config/default.yml.
       class UndefinedConfig < Base
+        class << self
+          include FileFinder
+        end
```

Comment:
```suggestion
        extend FileFinder
```

Reply:
`FileFinder` is a private API:
https://github.com/Earlopain/rubocop/blob/58eeb7a29ed6c35d111fb4f264c364261dc7ecbb/lib/rubocop/file_finder.rb#L7

Although this is an `InternalAffairs` cop, so the risk of issues arising might be low, it might be better to leave a comment in the source code noting that a private API is being used.

Reply:
I added a new test to `file_finder` to exercise it a bit more, I don't think it was being used in that way yet (looking for a file in a directory)